src/services/BackendService/BackendService.ts

```diff
@@ -27,16 +27,16 @@ export class BackendService implements BackendInterface {
     const message = `${request}:${Math.floor(new Date().getTime() / 1000 / 3600)}`;
     const hmac = hmac256(message, encHex.parse(this.hmacKey)).toString(encHex);
 
-    return downloadDiagnosisKeysFiles(`${this.retreiveUrl}/retrieve-day/${request}/${hmac}`);
+    return downloadDiagnosisKeysFile(`${this.retreiveUrl}/retrieve-day/${request}/${hmac}`);
```

Comment:
Just a typo for `retreiveUrl`: should be `retrieveUrl`. I think `period` also makes more sense than `request`; @burke do you have any other suggestions?

Also, `/retrieve-day` and `/retrieve-hour` don't exist any more; there's only one endpoint called `/retrieve` now. There's also a maximum of 15 requests / day according to [their docs](https://developer.apple.com/documentation/exposurenotification/setting_up_an_exposure_notification_server):

> An EN server can send multiple zip archives to the same client, but no more than 15 archives in a 24-hour period to any client.

We should retrieve only once every 2 (or maybe 3) hours to keep it easily under 15.

Reply:
ok thats a good point.

Reply:
Yeah, the `period` changes every 2 hours, since it's the latest even-numbered hour number (`Time.now.to_i / 3600 / 2) * 2`). The intention is to request new data at some point (up to) every 2 hours.